app/src/main/java/com/nitrr/ecell/esummit/ecellapp/models/LoginRVData.java

```diff
@@ -0,0 +1,26 @@
+package com.nitrr.ecell.esummit.ecellapp.models;
+
+import com.google.gson.annotations.SerializedName;
+
+import java.io.Serializable;
+
+public class LoginRVData implements Serializable {
```

Comment:
Need a more descriptive name for this file.

Reply:
it is a data java file for recycler view of Login activity